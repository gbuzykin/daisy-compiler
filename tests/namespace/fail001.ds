namespace ns1 {
struct ns2 {};
namespace ns2 {}
}
