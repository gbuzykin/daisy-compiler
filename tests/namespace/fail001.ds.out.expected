./namespace/fail001.ds:1:1: debug: token
 1 | namespace ns1 {
   | ^~~~~~~~~
./namespace/fail001.ds:1:11: debug: id: ns1
 1 | namespace ns1 {
   |           ^~~
./namespace/fail001.ds:1:15: debug: token
 1 | namespace ns1 {
   |               ^
./namespace/fail001.ds:2:1: debug: token
 2 | struct ns2 {};
   | ^~~~~~
./namespace/fail001.ds:2:8: debug: id: ns2
 2 | struct ns2 {};
   |        ^~~
./namespace/fail001.ds:2:12: debug: token
 2 | struct ns2 {};
   |            ^
./namespace/fail001.ds:2:13: debug: token
 2 | struct ns2 {};
   |             ^
./namespace/fail001.ds:2:14: debug: token
 2 | struct ns2 {};
   |              ^
./namespace/fail001.ds:3:1: debug: token
 3 | namespace ns2 {}
   | ^~~~~~~~~
./namespace/fail001.ds:3:11: debug: id: ns2
 3 | namespace ns2 {}
   |           ^~~
./namespace/fail001.ds:3:15: debug: token
 3 | namespace ns2 {}
   |               ^
./namespace/fail001.ds:3:11: error: redefinition of `ns2` as different kind of entity
 3 | namespace ns2 {}
   |           ^~~
./namespace/fail001.ds:2:8: note: previous definition is here
 2 | struct ns2 {};
   |        ^~~
./namespace/fail001.ds:3:16: debug: token
 3 | namespace ns2 {}
   |                ^
./namespace/fail001.ds:4:1: debug: token
 4 | }
   | ^
./namespace/fail001.ds: info: warnings 0, errors 1
